prdoc/pr_8887.prdoc

```diff
@@ -0,0 +1,9 @@
+title: '`fatxpool`: added mortal transactions integration test '
```

Comment:
TODO: a prdoc shouldn't be needed by this PR which adds just tests. However, even I pick the label for no crates should be published, check semver still fails detecting changes in the crate. Not sure if I am supposed to do something else, to check with release team.